### PR TITLE
fix: open logo link in new tab

### LINKED ISSUE #210: Clicking Anime Skip logo in player breaks it
### Error in player
image

### Error in console
image

### Resolution
My assumption is that the expected behavior is to open [anime-skip.com](https://www.anime-skip.com/) in a new tab - is this true?

### PR DESCRIPTION
**Pull request by logiczsniper**
Fixes #210

### CODE REVIEW

File: src/content-scripts/player/components/EpisodeInfo.vue [hunk 1]
```diff
@@ -1,7 +1,13 @@
 <template>
   <div class="EpisodeInfo as-space-y-4" :class="{ 'as-visible': isShowing }">
     <div class="as-flex as-flex-row as-items-center as-space-x-3">
-      <a v-if="showThemeLogo" class="as-pr-1" href="https://anime-skip.com">
+      <a
+        v-if="showThemeLogo"
+        class="as-pr-1"
+        href="https://anime-skip.com"
+        target="_blank"
+        :onclick="event => event.stopPropagation()"
```
Comment (aklinker1) [inline src/content-scripts/player/components/EpisodeInfo.vue:9]: Is this needed?

Comment (logiczsniper) [reply src/content-scripts/player/components/EpisodeInfo.vue:9]: This stops the video from playing when you click the logo which makes sense to me as the new tab opens, you don't really want the anime you were watching to start playing... but if you want the video to play I can remove it no problem!

Comment (aklinker1) [reply src/content-scripts/player/components/EpisodeInfo.vue:9]: Ahh, I see, it stops the pause. Also, with vue you can do `@click.stop` instead.

```suggestion
        @click.stop
```

Comment (logiczsniper) [reply src/content-scripts/player/components/EpisodeInfo.vue:9]: 😮 that's awesome, thanks!!

**Review by aklinker1 — APPROVED** [review]

**Comment by aklinker1** [comment]
Gonna merge this after it passes, then I'll do the next release! Several bug fixes were needed after that last one lol

**Pull request merged**